demisto_sdk/commands/pre_commit/README.md

```diff
@@ -18,6 +18,65 @@ It utilizes the [pre-commit](https://github.com/pre-commit/pre-commit) infrastru
 ### Automatically, as a git hook
 * Create a [git hook](https://git-scm.com/book/en/v2/Customizing-Git-Git-Hooks) that calls `demisto-sdk pre-commit`.
 
+**Arguments**:
+* **-i, --input, --files**
+The paths to run pre-commit on. May pass multiple paths.
+* **--staged-only**
+Whether to run only on staged files.
+* **--commited-only**
+Whether to run on committed files only.
+* **-g, --git-diff**
+Whether to use git to determine which files to run on.
+* **--prev-ver**
+The previous version to compare against. If not provided, the previous version will be determined using git.
+* **-a, --all-files**
+Whether to run on all files.
+* **--mode**
+Special mode to run the pre-commit with.
```

Comment:
this isnt clear enough IMO. we need docs for how to use properly (and this should give a link)

Reply:
There's a whole section for modes